editorconfig.go

```diff
@@ -122,7 +123,11 @@ func (e *Editorconfig) GetDefinitionForFilename(name string) (*Definition, error
 		}
 
 		if !strings.HasPrefix(name, "/") {
```

Comment:
this will probably break if you give a full path instead of a relative one, e.g. `C:\Users\me\foo.toml`